ironfish-cli/src/commands/ceremony.ts

```diff
@@ -0,0 +1,170 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { contribute } from '@ironfish/rust-nodejs'
+import { ErrorUtils, PromiseUtils } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import axios from 'axios'
+import fsAsync from 'fs/promises'
+import path from 'path'
+import { pipeline } from 'stream/promises'
+import { IronfishCommand } from '../command'
+import { DataDirFlag, DataDirFlagKey, VerboseFlag, VerboseFlagKey } from '../flags'
+import { CeremonyClient } from '../trusted-setup/client'
+
+export default class Ceremony extends IronfishCommand {
+  static description = 'Contribute randomness to the Iron Fish trusted setup'
+
+  static flags = {
+    [VerboseFlagKey]: VerboseFlag,
+    [DataDirFlagKey]: DataDirFlag,
+    host: Flags.string({
+      parse: (input: string) => Promise.resolve(input.trim()),
+      default: '127.0.0.1',
```

Comment:
Should add a default here. I've added `ceremony.ironfish.network` to DNS

Reply:
I'll switch this default over to that subdomain then.